include/ur_client_library/log.h

```diff
@@ -28,6 +28,11 @@
 #define LOG_ERROR CONSOLE_BRIDGE_logError
 #define LOG_FATAL CONSOLE_BRIDGE_logError
 
+#elif defined(ISAAC_BUILD)
+#include "engine/core/logger.hpp"
+
+#define LOG_WARN(format, ...)
```

Comment:
Why is there only a define for `LOG_WARN` and why is it empty?

Reply:
I have added a commit where I define LOG_WARN to use LOG_WARNING defined in "engine/core/logger.hpp"
The other defines are declared within "engine/core/logger.hpp" file.

Reply:
That answers the second question. But what about the other log macros?

Reply:
LOG_DEBUG, LOG_INFO and LOG_ERROR are all defined inside "engine/core/logger.hpp", so it should be enough to just indclude "engine/core/logger.hpp".

LOG_FATAL is not defined inside the file, so I will fix that. 
The build wasn't failing, because LOG_FATAL isn't used anywhere in the library.

Reply:
And `engine/core/logger.hpp` is where?

It seems like this is requiring files from the ISAAC driver, which does not seem right.

Reply:
"engine/core/logger.hpp" is part of the Isaac SDK framework.